### PR TITLE
master: only one place for storage GVRs

### PR DESCRIPTION
**Pull request by sttts**
We were overwriting the pkg/cmd/server/origin/rest.StorageOptions values in master_config.go's BuildStorageFactory.

### CODE REVIEW

**Comment by sttts** [comment]
Fixes https://github.com/openshift/origin/pull/13653#discussion_r112765259

**Comment by sttts** [comment]
[test]

File: pkg/cmd/server/origin/rest/storage_options.go [hunk 1]
```diff
@@ -46,33 +46,33 @@ func StorageOptions(options configapi.MasterConfig) (restoptions.Getter, error)
...
-			{"route.openshift.io", "v1", "routes"},
-			{"network.openshift.io", "v1", "netnamespaces"},
-			{"network.openshift.io", "v1", "hostsubnets"},
-			{"network.openshift.io", "v1", "clusternetworks"},
-			{"network.openshift.io", "v1", "egressnetworkpolicies"},
-			{"template.openshift.io", "v1", "templates"},
-			{"user.openshift.io", "v1", "groups"},
-			{"user.openshift.io", "v1", "users"},
-			{"user.openshift.io", "v1", "identities"},
+		// storage versions: use legacy group name "" for all resources that existed when apigroups were introduced
```
Comment (mfojtik) [inline pkg/cmd/server/origin/rest/storage_options.go]: maybe a small const will make this better? `LegacyGroup` or something :-)

Comment (sttts) [inline pkg/cmd/server/origin/rest/storage_options.go]: done

**Comment by sttts** [comment]
@mfojtik ptal

**Comment by sttts** [comment]
re[test]

**Comment by sttts** [comment]
re[test]

**Comment by sttts** [comment]
@mfojtik ptal

**Comment by mfojtik** [comment]
LGTM

**Comment by sttts** [comment]
[merge]

**Pull request merged**